.github/workflows/release.yml

```diff
@@ -0,0 +1,60 @@
+name: release-version
+
+on:
+  push:
+    tags:
+      - v*.*.*
+
+jobs:
+  release-version:
+    name: release-version
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v2.3.4
+        with:
+          fetch-depth: "0"
+      - run: git fetch origin +refs/tags/*:refs/tags/*
+      - name: Use Node.js 14
+        uses: actions/setup-node@v2.1.4
+        with:
+          node-version: 14
+          registry-url: https://registry.npmjs.org/
+      - name: Install dependencies
+        run: "bash setup_${{ runner.os }}.sh"
+      - run: npm ci
+      - run: npm run build
+      - name: Run unit/integration tests
+        uses: GabrielBB/xvfb-action@v1
+        with:
+          run: "bash run_${{ runner.os }}.sh"
+      - name: Send results to SonarCloud
+        if: runner.os == 'Linux'
+        uses: SonarSource/sonarcloud-github-action@v1.4
+        env:
+          GITHUB_TOKEN: ${{ secrets.GITHUB_TOKEN }}
+          SONAR_TOKEN: ${{ secrets.SONAR_TOKEN }}
```

Comment:
```suggestion
```
same as for cd. I think we can omit QA analysis on release builds as no code change will happen on master or tags.